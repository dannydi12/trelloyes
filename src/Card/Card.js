import React from 'react';
import './Card.css';

function Card(props) {
  
}

export default Card;