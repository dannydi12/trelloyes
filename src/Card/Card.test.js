import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer'
import Card from './Card';

describe('Smoke test', () => {

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Card />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('matches snapshot', () => {
    const tree = renderer.create(<Card title='homework' content='do it'/>).toJSON();
    expect(tree).toMatchSnapshot();
  });


})