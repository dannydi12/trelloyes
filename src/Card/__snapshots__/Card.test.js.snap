// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Smoke test matches snapshot 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    homework
  </h3>
  <p>
    do it
  </p>
</div>
`;
